common/src/chain/tokens/rpc.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use serialization::Decode;
+use serialization::Encode;
+
+// use crate::primitives::VersionTag;
+use crate::{
+    chain::{Block, Transaction},
+    primitives::{Amount, Id},
+};
+
+use super::TokenId;
+
+#[derive(Debug, Clone, Encode, Decode, serde::Serialize, serde::Deserialize)]
+pub struct RPCTokenInfoV1 {
+    // TODO: Should we use here VersionTag?
+    pub version: u32,
```

Comment:
What does this version represent?

Reply:
As you explain in the previous comment that there is no need to add versioning, I'm going to remove this field. 

Before that, I thought that might be we have to use versioning, and was doubts about should we use VersionTag like it does in BlockHeader, or like I did with token data. This is why I left here the todo.